build.sbt

```diff
@@ -698,7 +698,7 @@ lazy val flinkExecutor = (project in flink("executor")).
         "org.apache.flink" % "flink-streaming-java" % flinkV % "provided",
         "org.apache.flink" % "flink-runtime" % flinkV % "provided",
         "org.apache.flink" % "flink-statebackend-rocksdb" % flinkV % "provided",
-        "org.apache.flink" % "flink-metrics-dropwizard" % flinkV,
```

Comment:
Are you sure about this change? Will it work correctly with test from file mechanism which run embedded Flink on the designer side?

Reply:
it looks like it doesn't. I've read in comment is build.sbt that we use bundled metrics on flink. I assumed it meant that they should be provided, so we don't have to bring our own library in general as it is provided on ververica.

Reply:
I've recently been able to fix all tests